### ISSUE TITLE
JP SEO feature vs wpcom SEO feature confusion

### ISSUE THREAD

**Issue by StefMattana** [Automattic/wp-calypso#12421]
User purchased the WordPress.com Pro plan thinking of massively using the Jetpack SEO feature. 
<a href="https://en.support.wordpress.com/seo-tools/">As explained on this page</a>, only wpcom Business plans have in-house SEO features, while the Jetpack SEO tools are available on the wporg / plugin option. Right?

Yet, the user made a good point. If you click on the `i` in any of our wpcom plans (both on Calypso and on the <a href="https://wordpress.com/pricing/">Pricing page</a>, the popup window says the following: 
<img width="585" alt="screen shot 2017-03-22 at 15 30 04" src="https://cloud.githubusercontent.com/assets/11662998/24208188/db5f637c-0f1a-11e7-994a-b6ecd90c0604.png">

I think this is confusing for the user and we should reword it to make sure users don't think there's the Jetpack SEO feature included. 

For the record, the user was after a plan closer to the Business than the Pro one, and was disappointed about it. Since they provided a good feedback to them, I upgraded them to Business for one year. 

Hope that's clear enough! Please ping me if you need any more info about it :)

(internal ref: 259704-hc)

#context #perspective #user-report

**Comment by designsimply** [comment]
> [As explained on this page](https://en.support.wordpress.com/seo-tools/), only wpcom Business plans have in-house SEO features, while the Jetpack SEO tools are available on the wporg / plugin option. Right?

My interpretation of the first paragraph on the SEO Tools page at https://en.support.wordpress.com/seo-tools/ is that only WordPress.com Business and Jetpack Pro plans include advanced SEO tools.

> Optimize your site for search engines and social media by taking advantage of our SEO tools. This feature is only available in the WordPress.com Business Plan and the Jetpack Professional Plan for your self-hosted site.

The Pricing page at https://wordpress.com/pricing/ mentions "SEO" (meaning things such as site verification and XML sitemaps) in the (i) tooltip and has a whole separate line item for "SEO Tools" (meaning things such as search engine previews and custom meta such as titles and descriptions).

![screen shot 2017-03-22 at wed mar 22 10 56 21 am](https://cloud.githubusercontent.com/assets/1119271/24211108/b36bbed0-0ef0-11e7-91fd-a5690c2b1c63.png)
<sup>Seen at https://wordpress.com/pricing/ using Chrome 56.0.2924.87 on Mac OS X 10.12.3.</sup>

(For reference) pricing page "Jetpack Essential Features" info text:

> Jetpack is a powerful plugin that includes SEO, spam protection, social sharing, site stats, and more.

(For reference) pricing page "SEO Tools" info text: 

> Adds tools to enhance your site's content for better results on search engines and social media.

The Jetpack Professional Plan page at https://jetpack.com/install/professional/ also has a nice comparison graph showing the differences:

![screen shot 2017-03-22 at wed mar 22 11 15 48 am](https://cloud.githubusercontent.com/assets/1119271/24211207/f2ca6bc6-0ef0-11e7-8abd-c1ae17bbb777.png)
<sup>Seen at https://jetpack.com/install/professional/ using Chrome 56.0.2924.87 on Mac OS X 10.12.3.</sup>

There is only a very limited space to mention SEO features on the main Pricing page, especially in the (i) bubble! Because of the limited space in that area and because this issue is based on only one user report, I am going to recommend closing this issue for now. Improving the related support SEO Tools page at https://en.support.wordpress.com/seo-tools/ can be done separate from `wp-calypso` if you think that would help make things more clear for users. It's possible that SEO itself can be difficult to define and is not well-understood by many in the general population and that's when we need to work through explaining the options to them when misunderstandings come up such as in the chat session you mentioned.

If anyone has specific suggestions for changing the copy in the info tooltips on the Pricing page at  https://wordpress.com/pricing/ please re-open this issue and include suggested copy.

**Comment by tmmbecker** [comment]
Just got a ticket from a user who purchased a Personal Plan and is looking for SEO, with a direct quote of the info in the (i) bubble. 

for internal reference 3123329-t

**Comment by aportbury** [comment]
I also have a ticket from a user who purchased the personal plan with the understanding that it provided SEO tools. 

Reference: 3325711-t

**Comment by StefMattana** [comment]
Has this issue got sorted out?

**Comment by tmmbecker** [comment]
Still seeing that same information in the tip. Let's keep this open.

**Comment by tmmbecker** [comment]
another confused and frustrated user here 1112291-zen

**Comment by sarahgreenlee** [comment]
Another report of confusion and frustration 1108690-zen 

Is there any way to clarify for users what is exactly included in the Jetpack Essentials Features? The comparison chart jetpack.com/features/comparison isn't clear about which plan is considered Jetpack Essential Features. Could we add a page to our WordPress.com support about what's included from Jetpack in the essential features to make it clear that it's not the Jetpack Premium or Professional features and link to that from the pricing pages?
